test/testenv/verificationutils.go

```diff
@@ -490,17 +490,47 @@ func VerifySplunkServerConfSecrets(deployment *Deployment, testenvInstance *Test
 	}
 }
 
+// VerifySplunkInputConfSecrets Compare secret value on passed in map to value present on input.conf for given indexer or standalone pods
+// Set match to true or false to indicate desired +ve or -ve match
+func VerifySplunkInputConfSecrets(deployment *Deployment, testenvInstance *TestEnv, verificationPods []string, data map[string][]byte, match bool) {
+	secretName := "hec_token"
+	for _, podName := range verificationPods {
+		if strings.Contains(podName, "standalone") || strings.Contains(podName, "indexer") {
+			found := false
+			testenvInstance.Log.Info("Key Verificaton", "Pod Name", podName, "Key", secretName)
+			stanza := SecretKeytoServerConfStanza[secretName]
+			_, value, err := GetSecretFromInputConf(deployment, podName, testenvInstance.GetName(), "token", stanza)
+			gomega.Expect(err).To(gomega.Succeed(), "Secret not found in conf file", "Secret Name", secretName)
+			comparsion := strings.Compare(value, string(data[secretName]))
+			if comparsion == 0 {
+				testenvInstance.Log.Info("Secret Values on input.conf Match", "Match Expected", match, "Pod Name", podName, "Secret Key", secretName, "Given Value of Key", string(data[secretName]), "Key Value found", value)
+				found = true
+			} else {
+				testenvInstance.Log.Info("Secret Values on input.conf DONOT MATCH", "Match Expected", match, "Pod Name", podName, "Secret Key", secretName, "Given Value of Key", string(data[secretName]), "Key Value found", value)
```

Comment:
Nit: Does it need to be Log.Error()?

Reply:
Error is handled a line above. This area just compares values. So no error is generated

Reply:
@sgontla the method is used for +ve as well as -ve match. Depending on the called a mismatch might be a success.